### DISCUSSION TITLE
Unable to access DP Library

### DISCUSSION THREAD

**Discussion by RuiYang1999** [deepmodeling/deepmd-kit#2465] [Q&A] [answered]
If I'm right the URL to DP Library is https://dplibrary.deepmd.net/, but I'm unable to access it.

**Comment by AnguseZhang** [comment, marked as answer]
@RuiYang1999 
Hi, RuiYang. 

As a hub for models and data related to Deep Potential, DP Library with contributors has played a vital role in promoting the community and advancing research in this area.

However, we gradually recognized that the current website lacks flexibility for accommodating various data and has brought sustaining inconveniences for users, which led us to endeavour a better solution. 

[AI & Science Square (AIS-Square)](https://www.aissquare.com/) serves AI4S researchers with three main modules, naming models, datasets and workflows. With its user-friendly interface, AIS-Square allows users to conveniently access accumulated data and share it with others. We believe that this platform will offer a better solution for the community's needs, and we are looking forward to getting along with you in this new venture.

As a part of our efforts to streamline resources, we decided to migrate all content from DP Library to [AI & Science Square (AIS-Square)](https://www.aissquare.com/) in a proper and thorough manner. As a result, DP Library will no longer be updated or maintained. We assure that the migration will be a meticulous process:
- With a guarantee that all content, including datasets and models, was migrated to corresponding entries on AIS-Square.
- To facilitate a smooth transition, we created an introduction to the AIS-Square entry, with references to the original DP Library entry and its authors. 
- Additionally, we provided an automatic redirect (with an explanation) from the DP Library entry to the corresponding AIS-Square entry. This will ensure that users can seamlessly access the migrated content on AIS-Square. If it fails, we'll figure out what happens.  


Hope that you can find the data you need in AIS-Square. 


Best, 

Yuzhi

**Reply by RuiYang1999** [reply]
Thank you very much for your answer! I'll check it out.
